src/Entity/Game.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\GameRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @ORM\Entity(repositoryClass=GameRepository::class)
+ */
+class Game
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Ship", mappedBy="game")
+     */
+    private $ships;
+
+    public function __construct()
+    {
+        $this->ships = new ArrayCollection();
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getShips(): Collection
+    {
+        return $this->ships;
+    }
+
+    public function addShip(Ship $ship): self
+    {
+        $this->ships[] = $ship;
+
+        return $this;
+    }
+
+    public function getShipAt($x, $y)
```

Comment:
This must be placed in another file or service. (can be done in another PR. In this case, need to be defined in a @TODO instruction)

Moreover, you need to add a description and typing.